kernel/shop/customerorderview.php

```diff
@@ -16,6 +16,13 @@
 $tpl = eZTemplate::factory();
 
 $Email = urldecode( $Email );
+
+// workaround because it seems not possible to get an urlencoded "+" character accross $Params
+if ( $http->hasGetVariable( "email" ) )
+{
+    $Email = $http->getVariable( "email" );
+}
+
```

Comment:
But, if i'm not wrong, here you are remove Email Param from the url, right? i mean, [this](https://github.com/ezsystems/ezpublish-legacy/blob/master/kernel/shop/module.php#L112) is not valid as you won't have Email as Param anymore